src/core/include/openvino/op/roi_align.hpp

```diff
@@ -81,6 +81,88 @@ class OPENVINO_API ROIAlign : public Op {
     PoolingMode m_mode;
 };
 }  // namespace v3
+
+namespace v9 {
+class OPENVINO_API ROIAlign : public Op {
+public:
+    OPENVINO_OP("ROIAlign", "opset9", op::Op, 9);
+    BWDCMP_RTTI_DECLARATION;
+    enum class PoolingMode { AVG, MAX };
+    enum class AlignedMode { ASYMMETRIC, TF_HALF_PIXEL_FOR_NN, HALF_PIXEL };
+
+    ROIAlign() = default;
+    /// \brief Constructs a ROIAlign node matching the ONNX ROIAlign specification
+    ///
+    /// \param input           Input feature map {N, C, H, W}
+    /// \param rois            Regions of interest to pool over
+    /// \param batch_indices   Indices of images in the batch matching
+    ///                        the number or ROIs
+    /// \param pooled_h        Height of the ROI output features
+    /// \param pooled_w        Width of the ROI output features
+    /// \param sampling_ratio  Number of sampling points used to compute
+    ///                        an output element
+    /// \param spatial_scale   Spatial scale factor used to translate ROI coordinates
+    /// \param mode            Method of pooling - 'avg' or 'max'
+    /// \param aligned_mode    Method of coordinates alignment - 'asymmetric', 'tf_half_pixel_for_nn' or 'half_pixel'
+    ROIAlign(const Output<Node>& input,
+             const Output<Node>& rois,
+             const Output<Node>& batch_indices,
+             const int pooled_h,
+             const int pooled_w,
+             const int sampling_ratio,
+             const float spatial_scale,
+             const std::string& mode,
+             const std::string& aligned_mode = "asymmetric");
+
+    ROIAlign(const Output<Node>& input,
+             const Output<Node>& rois,
+             const Output<Node>& batch_indices,
+             const int pooled_h,
+             const int pooled_w,
+             const int sampling_ratio,
+             const float spatial_scale,
+             const PoolingMode mode,
+             const AlignedMode aligned_mode = AlignedMode::ASYMMETRIC);
+
+    void validate_and_infer_types() override;
+    bool visit_attributes(AttributeVisitor& visitor) override;
+    std::shared_ptr<Node> clone_with_new_inputs(const OutputVector& new_args) const override;
+
+    int get_pooled_h() const {
+        return m_pooled_h;
+    }
+    int get_pooled_w() const {
+        return m_pooled_w;
+    }
+    int get_sampling_ratio() const {
+        return m_sampling_ratio;
+    }
+    float get_spatial_scale() const {
+        return m_spatial_scale;
+    }
+    PoolingMode get_mode() const {
+        return m_mode;
+    }
+    AlignedMode get_aligned_mode() const {
+        return m_aligned_mode;
+    }
+    OPENVINO_SUPPRESS_DEPRECATED_START
+    bool evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const override;
```

Comment:
Do we need evaluate for this operation?

Reply:
Hi, Ilya: as we synced offline, now have removed the evaluate function for ROIAlign_9